src/dotnet/Telemetry.cs

```diff
@@ -45,6 +70,8 @@ public Telemetry()
                 _commonProperties.Add(OSPlatform, runtimeEnvironment.OperatingSystemPlatform.ToString());
                 _commonProperties.Add(RuntimeId, runtimeEnvironment.GetRuntimeIdentifier());
                 _commonProperties.Add(ProductVersion, Product.Version);
+                _commonProperties.Add(CITestMachine, isCITestMachine.ToString());
```

Comment:
This has been refactored to just TestMachine for now.

Telemetry can be turned off by anyone at anytime for any reason by simply setting the env variable DOTNET_CLI_TELEMETRY_OPTOUT=1